MRG: Refactor/bids compatible with MNE-BIDS/MNE-Python, BidsRun Func, Example Setup and Reading (#8)
adam2392/eegio: README.md

    numpy
    scipy
    scikit-learn
    pandas
    mne
    mne-bids
    pybids
    seaborn
    matplotlib
    pyedflib (deprecated)
    xlrd (deprecated)
    
See [INSTALLATION GUIDE](./docs/INSTALLATION.md)

## Intended Users / Usage

Epilepsy researchers dealing with EEG data compliant with BIDS and MNE formats. Anyone with human patient EEG data.  
See example and docs for info on how to format this.

* https://github.com/bids-standard/bids-specification/blob/master/src/04-modality-specific-files/04-intracranial-electroencephalography.md
* https://github.com/bids-standard/bids-starter-kit/wiki/The-BIDS-folder-hierarchy
    
### Setting Up the BIDS Directory and Reading EEG Data From EDF/FiF
These are just lightweight wrappers of MNE/pyedflib reading to load in EDF/FiF data
easily, so that raw EEG ts are readily accessible in Python friendly format. We provide
an example that was built off of the examples in MNE-BIDS.
See [example](./examples/read_eeg_from_bids.py).
    
For more info, see tutorials and documentation.

# Contributing

> We welcome contributions from anyone. Our [issues](https://github.com/adam2392/eegio/issues) page is full of places we could use help! 

Reviewer: ```suggestion
We welcome contributions from anyone. Our [issues](https://github.com/adam2392/eegio/issues) page is a great place for suggestions.
```

> We welcome contributions from anyone. Our [issues](https://github.com/adam2392/eegio/issues) page is full of places we could use help! 

Reviewer: ```suggestion
We welcome contributions from anyone. Our [issues](https://github.com/adam2392/eegio/issues) page is a great place for suggestions!
```
